js/network.js

```diff
@@ -340,57 +217,31 @@ function initalizeNetwork() {
     networkNodeDrag = false;
   }
 
-  if (visMode == "dynamic") {
-    networkGravity =  0.05;
-    netwokrLinkLength = 500;
-    networkLargeNodeLimit = 20;
-    netwokrCharge = -800;
-    networkStopTick = false;
-    networkNodeDrag = true;
-  }
-
-  force.gravity(networkGravity);
+  force.gravity(.4);
   force.linkStrength(function(d) {  return linkStrength(d);});
+  //force.linkStrength(1.0);
   force.linkDistance(netwokrLinkLength);
   force.charge(netwokrCharge);
-  force.chargeDistance(visWidth/2);
-  force.friction(0.4);
+ // force.chargeDistance(visWidth/2);
+  force.friction(0.6);
 
   if (vis == null) {
-    zoom = d3.behavior.zoom()
-      .translate([0,0])
-      .scale(0.99)
-      .scaleExtent([0.25,6])	//how far it can zoom out and in
-      .on("zoom", redraw);
 
     vis = d3.select("#network").append("svg:svg")
       .attr("width", $("#network").width() - 10)
       .attr("height", $("#network").height() - 10)
-      .append('svg:g')
-      .call(zoom)//.call(d3.behavior.zoom().scaleExtent([0.25, 6]).on("zoom", redraw)) //.call(d3.behavior.zoom().on("zoom", redraw))
       .append('svg:g');
+      //.call(zoom);//.call(d3.behavior.zoom().scaleExtent([0.25, 6]).on("zoom", redraw)) //.call(d3.behavior.zoom().on("zoom", redraw))
 
     vis.append('svg:rect')
-      .attr('width', $("#network").width() + 1000)
-      .attr('height', $("#network").height() + 1000)
+      //.attr('width', $("#network").width() + 1000)
+      //.attr('height', $("#network").height() + 1000)
+	  .attr('width', $("#network").width())
+      .attr('height', $("#network").height())
       .attr('fill', 'white')
       .attr('id', 'zoomCanvas')
-      .style("cursor",  "url(menu/openhand.png)")
-      .on("mousedown", function() {
-
-        //the grabbing css rules do not work with web-kit, so specifiy the cursor hand and use the css for firefox.
-        d3.select("#zoomCanvas").style("cursor",  "url(menu/closedhand.png)");
-        d3.select("#zoomCanvas").attr("class","grabbing");
-      })
-      .on("mouseup", function() {
-        d3.select("#zoomCanvas").style("cursor",  "url(menu/openhand.png)");
-        d3.select("#zoomCanvas").attr("class","");
-      });
+      .style("cursor",  "url(menu/openhand.png)");
   }
-
-  vis.attr("transform",
```

Comment:
I wonder if this needs to stay. A similar command was required in the tick function to move nodes into place. Will keep an eye out for behavior anomalies that might point to this.